tutorials/docker/Dockerfile

```diff
@@ -1,27 +1,59 @@
-FROM rticom/connext-sdk:7.3.0-EAR as build-stage
+ARG RTI_LICENSE_AGREEMENT_ACCEPTED=false
+FROM ubuntu:20.04 as build-stage
+
+ARG RTI_LICENSE_AGREEMENT_ACCEPTED
+
+# Install the required packages
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install -y \
+    build-essential \
+    cmake \
+    curl \
+    git \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists
+
+# Install prerequisites
+RUN curl -sSL -o /usr/share/keyrings/rti-official-archive.gpg \
+    https://packages.rti.com/deb/official/repo.key
+RUN printf -- "deb [arch=%s, signed-by=%s] %s %s main\n" \
+    $(dpkg --print-architecture) \
+    /usr/share/keyrings/rti-official-archive.gpg \
+    https://packages.rti.com/deb/official \
+    $(. /etc/os-release && echo ${VERSION_CODENAME}) | \
+    tee /etc/apt/sources.list.d/rti-official.list >/dev/null
+
+# Install the RTI Connext Debian Package
+RUN export DEBIAN_FRONTEND=noninteractive \
+    RTI_LICENSE_AGREEMENT_ACCEPTED=${RTI_LICENSE_AGREEMENT_ACCEPTED} \
+    && apt-get update \
+    && apt-get install -y rti-connext-dds-7.3.0 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists
 
 # For armv8 uncomment the following line
 # ENV CONNEXTDDS_ARCH=armv8Linux4gcc7.3.0
```

Comment:
It would be nice to figure out the right CONNEXTDDS_ARCH based on the system's archicteure (for ex. with `uname -m`)

Reply:
I haven't tested this on armv8, this comment was leftover from how the file was before. If everything else looks good, I'll test this part in the lab to see if setting CONNEXTDDS_ARCH is needed and if so, using uname -m or some other cpu command